File: LandmarkBook/app/src/main/java/com/mustafauysal/landmarkbook/MainActivity.java
package com.mustafauysal.landmarkbook;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.ListView;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ListView listView = (ListView) findViewById(R.id.listView);

        ArrayList<String> landmarkNames = new ArrayList<String>();

        landmarkNames.add("Pisa");
        landmarkNames.add("Colloseo");
        landmarkNames.add("Eiffel");

        ArrayAdapter arrayAdapter = new ArrayAdapter(this, android.R.layout.simple_list_item_1, landmarkNames);

        listView.setAdapter(arrayAdapter);

    }
}

File: FunctionsAndClasses/app/src/main/java/com/musta/functionsandclasses/MainActivity.kt
package com.musta.functionsandclasses

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

@Suppress("UNUSED_PARAMETER")
class MainActivity : AppCompatActivity() {

    var age : Int? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        test()

        println(sum(5,10))


    }

    fun test() {
        val x = 5  * 4
        println(x)
    }

    fun sum (a: Int, b: Int): Int {
        return a + b
    }

    fun helloWorld(view: View) {
        textView.text = "Hello World"
    }

    fun makeSimpson(view: View) {

        val name = nameText.text.toString()

        if(!ageText.text.toString().equals("")) {
            age = ageText.text.toString().toInt()
        }

        val job = jobText.text.toString()

        val simpson = Simpson(name, age, job)

        textView.text = " Name: " + simpson.name + " Age:" + simpson.age + " Job:" + simpson.job


    }
}
